finishFeature/src/test/java/com/panhuk/finishfeature/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.panhuk.finishfeature
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+  @Test
+  fun addition_isCorrect() {
+    assertEquals(4, 2 + 2)
+  }
+}
```

Comment:
better to drop useless classes like this one or consider writing actual Unit Tests.

Reply:
No problem